bundles/net.tourbook/src/net/tourbook/database/TourDatabase.java

```diff
@@ -5506,6 +5594,71 @@ private void sqlStartup_UpgradeDerbyDatabase(final SplashManager splashManager)
       }
    }
 
+   private void updateDatabase(final Connection connection,
```

Comment:
`updateDb__2_Data` calls `updateDatabase` which calls `performTourDataUpdate_ForAllTours`

The code should be kept together by it's naming in an almost 10'000 line file.

I would suggest these naming:

`updateDatabase` -> `updateDb__3_Data_Concurrent`
`performTourDataUpdate_ForAllTours` -> `updateDb__4_Data_Concurrent_OneTour`

Now the methods are all together
![image](https://user-images.githubusercontent.com/1283445/163927138-4da4ceb2-147f-4a94-b4ad-9257dcc04c3d.png)

Reply:
> > The code should be kept together by it's naming in an almost 10'000 line file.
> 
> That's what makes it hard at times, some files are way too long (`TourData.java`, ....)

With more than 15+ years, a file gets longer but with the orange/green markers on the right side you have a very good overview where a field is read or written or methods are used, I use this feature very often

![image](https://user-images.githubusercontent.com/1283445/164158981-71f53985-c10f-4545-b4d9-094761110890.png)